generate-summary.py

```diff
@@ -11,10 +13,11 @@
 	total.append([0, list(tc)])
 	total[i][0] = i+1
 
-file = open("sezcist-20150606.csv", "rb")
-reader = csv.reader(file)
-for line in reader:
-	record = line[14]
+file = open(os.getcwd()+"\\assets\\data\\komunale.json", "rb")
```

Comment:
Why not just use a relative path like before? Would be much cleaner ...

Reply:
I did it this way because "komunale.json" is not located in the same folder as the python script in the original directory, but it's located under assets > data. We could also change it back to the relative path if that would be better, as I agree it is much cleaner

Reply:
Well yeah, "assets/data/komunale.json" is still a relative path.